panel/tests/widgets/test_select.py

```diff
@@ -500,7 +509,7 @@ def test_nested_select_partial_options_set(document, comm):
     select.options = {"Ben": []}
     assert select._widgets[0].value == 'Ben'
     assert select._widgets[0].visible
-    assert select.value == {0: 'Ben'}
+    assert select.value == {0: 'Ben', 1: None}
```

Comment:
I'm wondering if this is correct?

Reply:
Seems correct to me, there are two levels after all, even if one of them is empty.